### ISSUE TITLE
More reviewers & slack channel

### ISSUE THREAD

**Issue by shashken** [GoogleCloudPlatform/flink-on-k8s-operator#441]
Hey guys, I see we have a lot of open PRs & issues and looks like we will benefit from more reviewers.
@functicons As the repo owner, do you have any objection for this? Who can approve PRs now? what is the process to add approvers?
Also I noticed that the Slack channel link is broken (https://app.slack.com/client/T09NY5SBT/CQYSE926R)
Can it be fixed? or do we need to create a new slack channel?

**Comment by mvbakker** [comment]
I believe the Slack channel is a channel in the Kubernetes Slack? You can find me there, I've just mentioned this issue there. It has been pretty quiet there though.

**Comment by shashken** [comment]
@mvbakker The one I linked in this issue, it doesn't work

**Comment by mvbakker** [comment]
No, I see, which is weird because I generated it from Slack. Are you on the Kubernetes Slack? If you search for `flink operator` you can find a channel for it. That's where the link is supposed to bring you. I posted a message there last week, so far there doesn't seem to be much activity (probably because the link doesn't work).
